aptos-move/block-executor/src/counters.rs

```diff
@@ -128,56 +168,56 @@ pub static DEPENDENCY_WAIT_SECONDS: Lazy<Histogram> = Lazy::new(|| {
     .unwrap()
 });
 
-pub static PARALLEL_PER_BLOCK_GAS: Lazy<Histogram> = Lazy::new(|| {
-    register_histogram!(
-        "aptos_execution_par_per_block_gas",
```

Comment:
hmm.. are we breaking dashboards here?
cc @igor-aptos ?

Reply:
I added those counters. They are not used yet in the dashboards.